bot/pkg/bsky/bsky.go

```diff
@@ -92,3 +104,17 @@ func extractRKey(uri string) (string, error) {
 
 	return parts[len(parts)-1], nil
 }
+
+func createPost(URI string, content string, postType post.NYTContentType) (post.Post, error) {
+	if URI == "" || content == "" {
+		return post.Post{}, fmt.Errorf("empty content or uri. Content: %s, URI: %s", URI, content)
+	}
+	post := post.Post{
+		ID:      URI,
```

Comment:
non-blocking: We could potentially use the `rkey` for the Post.ID field (rather than the URI) for concision/deduplication

Reply:
Good idea, I'll refactor